test/lib/output/RunResultsProcessor.test.ts

```diff
@@ -378,3 +362,8 @@ ${getMessage(BundleName.RunOutputProcessor, 'output.writtenToOutFile', [fakeFile
 		});
 	});
 });
+
+function createResultsProcessor(display: Display, runOutputOptions: RunOutputOptions,
+								jsonReturnValueHolder: JsonReturnValueHolder): ResultsProcessor {
+	return (new ResultsProcessorFactoryImpl()).createResultsProcessor(display, runOutputOptions, jsonReturnValueHolder);
+}
```

Comment:
Although this test is called RunResultsProcessor.test.ts ... I decided to use the factory to create the run time results processor (which happens to be the composite). I did this because it'll make refactoring the RunResultsProcessor into smaller pieces in the future easier (where we can use this test to ensure we didn't break anything). Also it gives us the code coverage for the factory as well.